src/sql/workbench/browser/designer/designerPropertiesPane.ts

```diff
@@ -42,6 +49,16 @@ export class DesignerPropertiesPane {
 		return this._objectPath;
 	}
 
+	public set showDescription(value: boolean) {
```

Comment:
this property can be removed and the logic can be moved into setDescriptionText method

Reply:
we need this class property to remove the description for properties which dont have a description

Reply:
you can do the check in the setDescription method, if the property does not have a description, then hide it.